### PR TITLE
Add a version-check command

### LINKED ISSUE #19: Add script to check node version + npm version
We should add something that can be run in-front of every npm script. This should save some time in-case the node version is not up to snuff.

### PR DESCRIPTION
**Pull request by muffinresearch**
Fixes #19

### CODE REVIEW

File: .jscsrc [hunk 1]
```diff
@@ -1,4 +1,10 @@
 {
   "preset": "airbnb",
-  "excludeFiles": ['dist']
+  "excludeFiles": ['dist'],
+  "requireSpacesInsideObjectBrackets": false,
+  "requireCamelCaseOrUpperCaseIdentifiers": {
+    allExcept: [
+      "child_process",
```
Comment (tofumatt) [inline .jscsrc]: Super nitpick, but we should use trailing commas consistently if we're to use them.

(Personally I prefer not to as these configs are basically JSON and JSON would say "no trailing commas".)

But we should at least be consistent; this has a trailing comma, but the enclosing array and object don't.

File: .eslintrc [hunk 2]
```diff
@@ -1,3 +1,7 @@
 {
-  "extends": "airbnb"
+  "extends": "airbnb",
+  "rules": {
+    "object-curly-spacing": 0,
+    "no-var": 0,
```
Comment (tofumatt) [inline .eslintrc]: Same for trailing comma here.

**Comment by tofumatt** [comment]
r+wc

File: .jscsrc (hunk 1 above)
Comment (muffinresearch) [inline .jscsrc]: Yeah, annoyingly eslint and friends seem to use weird pseudo-JSON. I think it's mainly so you can have things like comments etc.

Comment (muffinresearch) [inline .jscsrc]: What we can do is add a test to make sure these are JSON parseable. Problem solved.

**Comment by tofumatt** [comment]
> What we can do is add a test to make sure these are JSON parseable. Problem solved.

Sounds great (included quote for posterity as your comments got hidden because of an outdated diff).

**Pull request merged**